### DISCUSSION TITLE
Root Access with Flsun V400 V1.4 image

### DISCUSSION THREAD

**Discussion by tita2000** [Guilouz/Klipper-Flsun-Speeder-Pad#131] [Show and tell]
Hello, I followed EtoTen U-Boot howto to gain root access with no success. After modifying the commands, I was able to gain root access. Please follow my link and read my comment.
[https://gist.github.com/EtoTen/5ad00314df4ec64ea018a6576fd59c2a?permalink_comment_id=4693659#gistcomment-4693659](https://gist.github.com/EtoTen/5ad00314df4ec64ea018a6576fd59c2a?permalink_comment_id=4693659#gistcomment-4693659)

Regards Timo

**Comment by CeanFritz** [comment]
Hi Timo,

I know this thread is old, but thought it was probably easiest to get your thoughts on this issue.

I was able to gain ssh access with the uboot method. However, I was following up with the Update dependencies guide here. After installing network manager, I cannot connect to my wifi network and ssh console takes much longer to finish loading. I believe it is from commands being ran from rc.local. 

Did you have any issues like this?

> [  OK  ] Started OpenBSD Secure Shell server.
> [   12.334981] rc.local[1969]: error: settime
> [   12.584086] rc.local[1969]: Successfully initialized wpa_supplicant
> [   12.600604] rc.local[1969]: ctrl_iface exists and seems to be in use - cannot override it
> [   12.624865] rc.local[1969]: Delete '/var/run/wpa_supplicant/wlan0' manually if it is not used anymore
> [   12.644594] rc.local[1969]: Failed to initialize control interface 'DIR=/var/run/wpa_supplicant GROUP=netdev'.
> [   12.664862] rc.local[1969]: You may have another wpa_supplicant process already running or the file was
> [   12.689198] rc.local[1969]: left by an unclean termination of wpa_supplicant in which case you will need
> [   12.708651] rc.local[1969]: to manually remove this file before starting wpa_supplicant again.
> [   12.724639] rc.local[1969]: nl80211: deinit ifname=wlan0 disabled_11b_rates=0
> [  OK  ] Started Tailscale node agent.
> [   13.166525] rc.local[1969]: OK
> [  337.737466] rc.local[1969]: PING 192.168.1.1 (192.168.1.1) 56(84) bytes of data.
> [  337.756753] rc.local[1969]: --- 192.168.1.1 ping statistics ---
> [  337.772720] rc.local[1969]: 1 packets transmitted, 0 received, +1 errors, 100% packet loss, time 0ms
> [  338.193012] rc.local[1969]: printer1 no 0
> [  338.204748] rc.local[1969]: USB-DISK the file exist
> [  338.220986] rc.local[1969]: reset_pad file exists
> [  338.303062] rc.local[1969]: down err
> [  339.185873] rc.local[2332]: resize2fs 1.45.5 (07-Jan-2020)
> [  339.201970] rc.local[2332]: Filesystem at /dev/mmcblk0p5 is mounted on /; on-line resizing required
> [  339.220778] rc.local[2332]: old_desc_blocks = 2, new_desc_blocks = 2
> [  339.229430] rc.local[2332]: The filesystem on /dev/mmcblk0p5 is now 3801211 (4k) blocks long.
> [  339.262821] rc.local[2335]: rm: cannot remove '.resize2fs': No such file or directory

**Reply by tita2000** [reply]
Hi, in case you did not manage to get your wifi running again, have you tried to remove the file mentioned 
[ 12.664862] rc.local[1969]: You may have another wpa_supplicant process already running or the file was
[ 12.689198] rc.local[1969]: left by an unclean termination of wpa_supplicant in which case you will need
[ 12.708651] rc.local[1969]: to manually remove this file before starting wpa_supplicant again.

I think it's your wifi config file that causes the trouble here. Although I haven't had a problem like yours yet.

Regards Timo
